commands

```diff
@@ -32,6 +32,7 @@ case "$1" in
     mkdir -p "$SERVICE_ROOT" || dokku_log_fail "Unable to create service directory"
     mkdir -p "$SERVICE_ROOT/data" || dokku_log_fail "Unable to create service data directory"
     mkdir -p "$SERVICE_ROOT/config" || dokku_log_fail "Unable to create service config directory"
+    curl "https://raw.githubusercontent.com/antirez/redis/${REDIS_IMAGE_VERSION:0:3}/redis.conf" > "$SERVICE_ROOT/config/redis.conf" || dokku_log_fail "Unable to download the default redis.conf to the config directory"
```

Comment:
Can you change this to:

```
${REDIS_IMAGE_VERSION:"0:3"}
```

Specifically, I'm looking to ensure we quote the right thing.